gordo_client/io.py

```diff
@@ -25,24 +21,24 @@ class ResourceGone(Exception):
     Used in our case for calling the server with a revision which is no longer used.
     """
 
-    pass
-
 
 class BadGordoRequest(Exception):
     """
     Represents a general 4xx bad request
     """
 
-    pass
+
+class BadGordoResponse(Exception):
+    """
+    Represents a general bad response (not json or model)
+    """
```

Comment:
Since we are passing `content` constructor should be looking like this:
```python
    def __init__(self, msg: str, content: bytes):
        self.content = content
        super().__init__(msg)
```